src/app/models/hris/EmployeeProfile/employeeType.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface EmployeeType{
+    id: number;
+    name: string;
+}
```

Comment:
Question: does this model not yet exist? Double check duplicate code for models.

Reply:
you are correct